doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -70,7 +70,7 @@ nRF9160
     * Added IPv6 support to all SLM services.
     * Added the GNSS service to replace the existing GPS test functionality.
     * Added the optional support of location services from nRF Cloud, like A-GPS, P-GPS, and cellular positioning.
-
+    * Added GPIO AT commands.
```

Comment:
Please don't change this file and wait for NCSv1.7.0 first

Reply:
Removed from release note.